README.md

```diff
@@ -969,20 +969,23 @@ meetingSession.audioVideo.realtimeSubscribeToAttendeeIdPresence(
 
 ### Monitoring and alerts
 
-**Use case 25.** Add an observer to receive video metrics. See `AudioVideoObserver` for more available metrics,
-such as WebRTC statistics processed by Chime SDK.
+**Use case 25.** Add an observer to receive WebRTC metrics processed by Chime SDK such as bitrate, packet loss, and bandwidth. See `AudioVideoObserver` for more available metrics.
 
 ```js
 const observer = {
-  videoSendHealthDidChange: (bitrateKbps, packetsPerSecond) => {
-    console.log(`Sending bitrate in kilobits per second: ${bitrateKbps} and ${packetsPerSecond}`);
+  metricsDidReceive: (clientMetricReport) => {
+    const metricReport = clientMetricReport.getObservableMetrics();
+
+    const videoUpstreamPacketPerSecond = metricReport.videoPacketSentPerSecond;
+    const videoUpstreamBitrate = metricReport.videoUpstreamBitrate;
+    const availableSendBandwidth = metricReport.availableOutgoingBitrate;
+    const availableRecvBandwidth = metricReport.availableIncomingBitrate;
+    const audioSpeakerDelayMs = metricReport.audioSpeakerDelayMs;
```

Comment:
nit: This could be simplified as:

```js
const {
  videoPacketSentPerSecond,
  videoUpstreamBitrate,
  availableOutgoingBitrate,
  availableIncomingBitrate,
  audioSpeakerDelayMs,
} = metricReport;
```

Reply:
Updated